mcsema/Arch/X86/Semantics/Bitcode/FXAM.cpp

```diff
@@ -4,7 +4,7 @@
 // FXAM semantics are adaptd from Remill
 DEFINE_SEMANTICS(FXAM)
 {
-  auto st0 = state->ST0;
+  auto st0 = DOUBLE(state->ST0);
```

Comment:
Does this have something with `long double`s on windows not actually being 80-bit floats?